application/src/main/kotlin/application/ExamplesCommand.kt

```diff
@@ -534,6 +538,36 @@ For example:
             })
         }
     }
+
+    @Command(
+        name = "transform",
+        mixinStandardHelpOptions = true,
+        description = ["Transform existing examples"]
+    )
+    class Transform: Callable<Unit> {
+        @Option(names = ["--contract-file"], description = ["Contract file path"], required = true)
+        lateinit var contractFile: File
+
+        @Option(names = ["--overlay-file"], description = ["Overlay file path"], required = false)
+        val overlayFile: File? = null
+
+        @Option(names = ["--examples-dir"], description = ["Directory where existing examples reside"], required = true)
+        lateinit var examplesDir: File
+
+        @Option(names = ["--only-mandatory-keys-in-payload"], description = ["Transform existing examples so that they contain only mandatory keys in payload"], required = false)
+        var allowOnlyMandatoryKeysInPayload: Boolean = false
```

Comment:
this should be implicitly set to true if given, we should not have to say --only-mandatory-keys-in-payload=true

Reply:
yeah makes sense, I've addressed this, also added a few simple tests for now.